packages/sitecore-jss-react/src/components/Placeholder.test.tsx

```diff
@@ -910,31 +880,24 @@ describe('PlaceholderMetadata', () => {
         },
         route: {
           name: 'main',
+          uid: 'root123',
           placeholders: {
-            main: [
-              {
-                uid: '123',
-                componentName: 'Layout',
-                placeholders: {
-                  header: [],
-                },
-              },
-            ],
+            main: [],
           },
         },
       },
     };
 
     const wrapper = shallow(
       <SitecoreContext componentFactory={componentFactory} layoutData={layoutData}>
-        <Placeholder name="header" rendering={layoutData.sitecore.route.placeholders.main[0]} />
+        <Placeholder name="main" rendering={layoutData.sitecore.route} />
       </SitecoreContext>
     );
 
     expect(wrapper.html()).to.equal(
       [
         '<div class="sc-jss-empty-placeholder">',
-        '<code type="text/sitecore" chrometype="placeholder" class="scpm" kind="open" id="header_123"></code>',
+        '<code type="text/sitecore" chrometype="placeholder" class="scpm" kind="open" id="main_root123"></code>',
```

Comment:
to avoid problems with this id value and other parameters you are testing, try using string literals instead of plaintext
i.e. for this one it could be `${layoutData.sitecore.route.name}_${layoutData.sitecore.route.uid}`

test inputs and expected outputs will always be in sync this way

Reply:
But the whole idea of adopting this format was to have plaintext.
Shouldn't matter since we are comparing it with the component output.
If there is a mismatch it would fail.

Reply:
This current PR was needed because there was a mismatch between input in layoutData and plaintext. The plaintext request was made to make the expected output more readable. Using string literals in _limited_ capacity (for ids for example) still brings the expected behavior across in readable way and makes the test more fault proof.

Reply:
No the current PR was needed(from the unit test POV) because of the way I was referencing placeholders from my mock layout data.
They should reference the root placeholder and I was always considering the nested placeholder.
There was nothing wrong with the outcome of any tests just the way it was setup.

My point is if there is anything incorrect with the plaintext then our test would fail. Right?